### DISCUSSION TITLE
Perguntas

### DISCUSSION THREAD

**Discussion by CaioCunha10** [CaioCunha10/TrollScriptWhatsApp-#9] [Q&A] [answered]
Tenho dúvidas de como adicionar novas falas de um filme famoso por completo.

**Comment by samuel-oldra** [comment, marked as answer]
Copie e cole as falas no meio do script

**Comment by marceloa1v3s** [comment]
coloca falas do filme do star wars

**Comment by CaioCunha10** [comment]
Achievements Galaxy Brain recebidos.
